drivers/adc/adc_stm32.c

```diff
@@ -1141,6 +1149,11 @@ static int adc_stm32_read(const struct device *dev,
 	struct adc_stm32_data *data = dev->data;
 	int error;
 
+	error = pm_device_runtime_get(dev);
```

Comment:
Ditto

Reply:
Do you mean integrate it in ` adc_stm32_read_async`